core/modules/system/views/views_handler_field_system_info.inc

```diff
@@ -0,0 +1,105 @@
+ * A handler to display module and theme settings stored in the info column.
+ *
+ * Provides access to several settings stored as a serialized array in the info
+ * column of the system table.
+ *
+ * @ingroup views_field_handlers
+ */
+class views_handler_field_system_info extends views_handler_field {
+  /**
+   * {@inheritdoc}
+   */
+  public function option_definition() {
+    $options = parent::option_definition();
+    $options['info_setting'] = array('default' => '');
+    $options['custom_label'] = array('default' => 'default');
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function options_form(&$form, &$form_state) {
+    parent::options_form($form, $form_state);
+
+    $form['custom_label']['#type'] = 'radios';
+    $form['custom_label']['#options'] = array(
+      'default' => t('Use setting label'),
+      'custom' => t('Custom label'),
+      'none' => t('No label'),
+    );
+    unset($form['custom_label']['#description']);
+
+    $form['custom_label']['#default_value'] = $this->options['custom_label'];
+    $form['label']['#states'] = array(
+      'visible' => array(
+        ':input[name="options[custom_label]"]' => array('value' => 'custom')
+      ),
+    );
+    $form['label']['#title'] = t('Custom label');
+
+    $form['info_setting'] = array(
+      '#type' => 'select',
+      '#title' => 'Field',
+      '#options' => $this->_getSettingOptions(),
+      '#default_value' => isset($this->options['info_setting']) ? $this->options['info_setting'] : '',
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render($values) {
+    $value = $this->get_value($values);
+    $info_array = unserialize($value);
+    $info_setting = $this->options['info_setting'];
+    if (!isset($info_array[$info_setting])) {
+      return '';
+    }
+    switch ($info_setting) {
+      case 'tags':
+      case 'dependencies':
+        return implode(', ', $info_array[$info_setting]);
+      default:
+        return $info_array[$info_setting];
+    } 
+  }
+
+  /**
+   * * Get this field's label based on the selected setting.
+   */
+  public function label() {
+    if ($this->options['custom_label'] === 'default') {
+      $setting_options = $this->_getSettingOptions();
+      $label = !empty($setting_options[$this->options['info_setting']]) ? $setting_options[$this->options['info_setting']] : '';
+      return $label;
+    }
+    elseif ($this->options['custom_label'] === 'custom' && isset($this->options['label'])) {
+      return $this->options['label'];
+    }
+    return '';
+  }
+
+  /**
+   * Helper function to return the option array.
+   */
+  protected function _getSettingOptions() {
+    return array(
+      'name' => t('Name'),
+      'description' => t('Description'),
+      'package' => t('Package'),
+      'configure' => t('Configuration path'),
+      'tags' => t('Tags'),
+      'version' => t('Version'),
+      'backdrop' => t('Backdrop version'),
+      'dependencies' => t('Dependencies'),
+    );
```

Comment:
Initially I thought of suggesting that we should use `ucfirst()` here, but then realized that by using specific strings in `t()`, we gain the following:
- these can be translatable
- we can customize the string that comes from the .info file (like in the case of `'configure' => t('Configuration path'),`)
...so good call here overall👍🏼

Having said that, can we please account for any other information that may be added in the .info files in the future? (by issues like https://github.com/backdrop-ops/backdropcms.org/issues/487 and/or https://github.com/backdrop/backdrop-issues/issues/5367). So the array with these "fixed" properties provided by core is perfectly fine, but can we account for other properties by converting their names to labels in this array via `ucfirst()`?

Reply:
Also, can we add `required` to the list of properties:
```suggestion
    return array(
      'name' => t('Name'),
      'description' => t('Description'),
      'package' => t('Package'),
      'configure' => t('Configuration path'),
      'tags' => t('Tags'),
      'version' => t('Version'),
      'backdrop' => t('Backdrop version'),
      'dependencies' => t('Dependencies'),
      'required' => t('Required'),
    );
```

Reply:
>   So the array with these default core-provided properties is perfectly fine, but can we account for other properties by converting their names to labels in this array via ucfirst()?

With the way these properties are declared, we can't. There is no central repository or definition of all possible info properties. You would have to scan all the info files for themes and modules to find out all the possible properties, as some properties are not required (e.g. tags. Some modules info files have tags, others don't). 

Instead, I've added an "Other" input, that allows you to type the name of any additional property not included in the array. This way you can possibly type new properties. And if necessary, this handler can be modified in the future to expand this array.

I've added several other improvements for flat array properties like tags and dependencies. You can now specify the separator, or order/unordered list. Also dependencies give you the option of outputting machine names or human labels.
 
But, since some properties can be multilevel arrays (like `spreadsheets`) I've accounted for that by outputting a `print_r` output. It doesn't look pretty, but it's better than just showing `Array` in the output. Things get just too complicated with multilevel arrays, or with arrays that use keys.